std_assert_test.go

```diff
@@ -0,0 +1,249 @@
+package objx_test
+
+import (
+	"fmt"
+	"reflect"
+	"runtime"
+	"testing"
+)
+
+type nonfatal struct{}
+type fatal struct{}
+
+var assert nonfatal
+var require fatal
```

Comment:
This idea is awesome !

I was like: WTF why removing testify imports everywhere. It won't work!

You got me, and I love the solution you found.